### PR TITLE
feature: keyboard shortcuts and tips

### PR DESCRIPTION
**Pull request by alex-krasn**

### CODE REVIEW

File: src/react/components/shell/hotKeysModal.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    <div className="header">
+                        <h4>HOT KEYS &amp; TIPS</h4>
+                        <p className="intro-text">
+                            Labeling tool allows a number of keyboard shortcuts to support accessibility and also sometimes make labeling easier and faster.</p>
+                    </div>
+                    <div className={"body"}>
+                        <h6 className="header">HOT KEYS LIST</h6>
+                        <ul className="hotkeys-list">
+                            {
+                                hotkeysItems.map((item) =>
```
Comment (stew-ro) [inline src/react/components/shell/hotKeysModal.tsx:103]: Could use UI Fabric basic list to achieve style similar to Microsoft Teams shortcut display https://developer.microsoft.com/en-us/fluentui#/controls/web/list 

image

File: src/common/localization/en-us.ts [hunk 2]
```diff
@@ -322,6 +322,69 @@ export const english: IAppStrings = {
...
+            description: "Can be used move the selection to the previous or the next word",
+        },
+        greaterAndLessThan: {
+            name: "greater-than and less-than",
+            ariaLabel: ",  or  .",
+            keys: {
+                key1: ",",
+                key2: ",",
+                },
+            description: "Can be used go to the previous or the next page in multi-pages documents",
```
Comment (stew-ro) [inline src/common/localization/en-us.ts:342]: Suggestion: remove "can be use to,"  similar to 
image

Comment (alex-krasn) [reply src/common/localization/en-us.ts:342]: Yeah, It feels redundant  - I took it from official readme, though. Let's check with Kun - it's easy change.

Comment (alex-krasn) [reply src/common/localization/en-us.ts:342]: @kunzms

Comment (kunzms) [reply src/common/localization/en-us.ts:342]: Yes, remove "Can be used "

**Pull request merged**